multiqc/modules/hicexplorer/hicexplorer.py

```diff
@@ -0,0 +1,184 @@
+        )
+
+        self.add_section(
+            name='Contact distance',
+            anchor='hicexplorer_contact_distance',
+            plot=self.hicexplorer_create_plot(keys_list_contact_distance, 'HiCExplorer: Categorization of reads')
+        )
+
+        self.add_section(
+            name='Read orientation',
+            anchor='hicexplorer_read_orientation',
+            plot=self.hicexplorer_create_plot(keys_list_read_orientation, 'HiCExplorer: Read orientation')
+        )
+
+    def parse_logs(self, f):
+        """Parse a given HiCExplorer log file from hicBuildMatrix."""
+        data = {}
+        for l in f.splitlines():
+            # catch empty lines
+            if len(l) == 0:
+                continue
+            s = l.split("\t")
+            data_ = []
+            # catch lines with descriptive content: "Of pairs used:"
+            for i in s[1:]:
+                if len(i) == 0:
+                    continue
+                try:
+                    i.replace('(', '')
+                    i.replace(')', '')
+                    i.replace(',', '')
+
+                    data_.append(float(i))
+                except ValueError:
+                    data_.append(i)
+            if len(data_) == 0:
+                continue
+            if s[0].startswith('short range'):
+                s[0] = 'short range'
+            elif s[0].startswith('same fragment'):
+                s[0] = 'same fragment'
+            s[0] = s[0].capitalize()
+            data[s[0]] = tuple(data_)
+        return data
+
+    def hicexplorer_basic_statistics(self):
+        """Create the general statistics for HiCExplorer."""
+        data = {}
+
+        for file in self.mod_data:
+            total_pairs = self.mod_data[file]['Pairs considered'][0]
+            data_ = {
+                'Pairs considered': self.mod_data[file]['Pairs considered'][0],
+
+                'Pairs used': self.mod_data[file]['Pairs used'][0] / total_pairs,
+                'Mapped': self.mod_data[file]['One mate unmapped'][0] / total_pairs,
+                'Min rest. site distance': self.mod_data[file]['Min rest. site distance'][0],
+                'Max rest. site distance': self.mod_data[file]['Max rest. site distance'][0],
+            }
+            data[self.mod_data[file]['File'][0]] = data_
+        headers = OrderedDict()
+
+        headers['Pairs considered'] = {
+            'title': 'M Number of pairs',
+            'shared_key': 'read_count'
+        }
+
+        headers['Pairs used'] = {
+            'title': '% Used pairs',
+            'max': 100,
+            'min': 0,
+            'modify': lambda x: x * 100
+        }
+        headers['Mapped'] = {
+            'title': '% Mapped',
+            'max': 100,
+            'min': 0,
+            'modify': lambda x: (1 - x) * 100,
+            'scale': 'RdYlGn'
+        }
+
+        headers['Min rest. site distance'] = {
+            'title': 'Min rest. site distance',
```

Comment:
It would be nice to make this title shorter (to give a narrower column) but with a longer tooltip description. Maybe something like this?

```python
'title': 'Min RE dist',
'descripton': 'Minimum restriction site distance (bp)'
```